src/components/StarRatingContainer.ts

```diff
@@ -0,0 +1,127 @@
+import { Component, createElement } from "react";
+import { StarRating } from "./StarRating";
+
+interface WrapperProps {
+    class?: string;
+    mxObject?: mendix.lib.MxObject;
+    style?: string;
+}
```

Comment:
Please check if `className `and `style` will always be there, I think they well be an empty string so in interface does not need be optional. Remove `?`